app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java

```diff
@@ -0,0 +1,122 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.google.firebase.firestore.FirebaseFirestore;
+
+/**
+ * Holds the view for seeing details on a book in the Requested tab
+ * The user will be able to interact with request options on the book
+ */
+//TODO: add status specific buttons, functions, and labels
+public class RequestDetailViewFragment extends Fragment
```

Comment:
RequestDetailViewFragment and BorrowDetailViewFragment are very similar to each other and MyBooksDetailViewFragment. Is there any way to inherit from MyBooksDetailViewFragment or perhaps make a RequestsDetailViewFragment and inherit from that?

Reply:
This is a good idea and will make using the filter really easy in the other activities